### ISSUE TITLE
Missing libraries of chopper

### ISSUE THREAD

**Issue by MohammadAliAmir** [wdecoster/chopper#8]
Hi @wdecoster, I installed the the binary of version 0.4.0 through chopper-linux.zip of that release. 
When I try to run it, it complains that there are some missing libraries required by chopper. Is that correct?
```
$ chopper
chopper: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by chopper)
chopper: /lib64/libc.so.6: version `GLIBC_2.25' not found (required by chopper)
chopper: /lib64/libc.so.6: version `GLIBC_2.18' not found (required by chopper)
chopper: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by chopper)
chopper: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by chopper)
chopper: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by chopper)
chopper: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by chopper)
chopper: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by chopper)
```
Can you give me all the required libraries/dependencies to use chopper?

Kind regards,
Mohammad Ali Amir

**Comment by MohammadAliAmir** [comment]
That looks fine now, thank you for the quick response!

**Comment by GuoYu-Peng** [comment]
Hell, v0.5.0 still have this problem?
```bash
$ ./chopper --help
./chopper: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./chopper)
./chopper: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./chopper)
./chopper: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./chopper)
```

**Comment by wdecoster** [comment]
Did you use the musl build?

**Comment by GuoYu-Peng** [comment]
I used binary file downloaded from the release.

**Comment by wdecoster** [comment]
Yes. Did you download the musl binary from the release?

**Comment by GuoYu-Peng** [comment]
ok, `chopper-musl.zip` works, thanks.

**Comment by kaanokay** [comment]
Hi,

1. Even if I used musl binary, I got the similar error: "/lib64/libm.so.6: version `GLIBC_2.29' not found".

2. In addition to that, installation of chopper via conda is problematic. When I tried to install it, I got following error: "Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve".

3. There is a bug in read length filtering, it does not filter reads that have length of interest.
